.github/CODEOWNERS

```diff
@@ -20,6 +20,7 @@
 /network_security/intercept/  @terraform-google-modules/pm2-team @terraform-google-modules/terraform-samples-reviewers
 /network_security/mirroring/  @terraform-google-modules/pm2-team @terraform-google-modules/terraform-samples-reviewers
 /privateca/                   @terraform-google-modules/dee-infra @terraform-google-modules/terraform-samples-reviewers
+/security_center_v2/          @terraform-google-modules/terraform-samples-reviewers
```

Comment:
```suggestion
/securitycenter/          @terraform-google-modules/terraform-samples-reviewers
```

Reply:
Don't accept this suggestion just yet; there's changes required from #854 that need to be adjusted, merging this will be a test of those new changes.